### PR TITLE
Wordsmithing

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,56 @@
...
 
-This project deploys your code to be deployed with [Capistrano](http://capistranorb.com/). If your project doesn't already use 
-Capistrano and have at least one deployment stage defined, that's a little beyond us here - go look at the Capistrano getting started 
-documentation and then come back here.
+This project deploys your code with [Capistrano](http://capistranorb.com/). If your project doesn't already use 
+Capistrano. Look to the Capistrano [getting started documentation](https://github.com/capistrano/capistrano#quick-start).
 
-You'll need to update your Hydra head (your application repository) to define a new stage corresponding to the environment you've just set up.  
-We usually give our stages name that help identify the server(s) they run on: e.g. demo, sandbox, stage, prod, etc.  Copy one of the existing stage files in your project (under `config/deploy`) to a new stage file.
+You'll need to update your Hydra application (your application repository) to define a new stage corresponding to the environment you've just set up.
```
Comment (acozine) [inline README.md:52]: The README is out of date. It's still possible to use this approach, but if you create the capistrano stage first and use the deploy role, ansible will deploy your code and restart the services for you. All you need to do after that is add the correct user keys so others can cap deploy from their dev environments.

File: README.md [hunk 2]
```diff
@@ -1,57 +1,56 @@
...
 ## Setup
 Before you can use this project, you must:
 
-1. clone this repo to your local workstation using: `git clone https://github.com/acozine/sufia-centos.git`  *NOTE: this script runs on your local system 
+1. Clone this repo to your local workstation using: `git clone https://github.com/acozine/sufia-centos.git`  * NOTE: this script runs on your local system 
    and connects to the server you want to configure, do not try to run it on the server directly*
-2. install [Ansible](http://docs.ansible.com/intro_installation.html) on your local system.
-3. continue with the EC2, Vagrant, or SSH section depending on your server type
+1. Install [Ansible](http://docs.ansible.com/intro_installation.html) on your local system.
+1. Continue with the Amaxon EC2, Vagrant, or SSH section depending on your server type
```
Comment (acozine) [inline README.md]: Amaxon?

File: README.md [hunk 3]
```diff
@@ -1,57 +1,56 @@
...
-To create an ec2 instance:
+To create an Amazon EC2 instance:
 
-1. create new vars/main.yml files in the launch_ec2 and ec2 roles  
-2. add your organization's AWS credentials there
-3. create a new vars/main.yml file in the services role  
-4. override any default variables you wish to change there (we definitely recommend overriding the postgresql database, user, and password settings)  
-5. run `ansible-playbook --private-key /path/to/your/keypair.pem ec2.yml` (if you encrypt your variables with ansible-vault, add `--ask-vault-pass`; if you are not using passwordless sudo, add `--ask-sudo-pass` or `-K`)  
-6. if the playbook fails, you can restart it at a particular task with `ansible-playbook --private-key /path/to/your/keypair.pem ec2.yml --start-at-task='rolename | taskname'`  
+1. create new vars/main.yml files in the `launch_ec2` and `ec2` roles
```
Comment (acozine) [inline README.md]: Initial cap here too?

**Comment by acozine** [comment]
If you'll fix the typo in Amazon and the one missing initial capitalization, then squash your commits, I'll approve the PR and then update the section about using Capistrano with info on the deploy role.

**Comment by acozine** [comment]
Also, thanks for going over the README so carefully.

**Comment by jcoyne** [comment]
@acozine fixed

File: README.md [hunk 4]
```diff
@@ -1,57 +1,56 @@
...
-3. edit the `hosts` file in this directory and replace the ip address for hydra-head with your server's IP
-4. edit the `vars:` section of the vanilla.yml file and enter data for your system - see the commented lines
-5. run `ansible-playbook -i hosts vanilla.yml` from the root directory of this repo on your local machine
-6. if the playbook fails, you can restart it at a particular task with `ansible-playbook -i hosts vanilla.yml --start-at-task='rolename | taskname'`  
+1. Create an installation user with passwordless sudo access on your server.  We called ours `centos`.  If you use a different name, edit the remote\_user entry in vanilla.yml
+1. Add the public key for your install user on the server in the users's .ssh/authorized\_keys file.  For help setting up public key authentication, see the [Centos Wiki](https://wiki.centos.org/HowTos/Network/SecuringSSH#head-9c5717fe7f9bb26332c9d67571200f8c1e4324bc) 
+1. these scripts assume that you have a separate virtual drive available for the application installation at `/dev/sdc` (assuming `/dev/sda` is your main system drive and `/dev/sdb` is your swap drive).
```
Comment (acozine) [inline README.md:41]: initial cap here?

File: README.md [hunk 5]
```diff
@@ -1,57 +1,56 @@
...
+1. Create an installation user with passwordless sudo access on your server.  We called ours `centos`.  If you use a different name, edit the remote\_user entry in vanilla.yml
+1. Add the public key for your install user on the server in the users's .ssh/authorized\_keys file.  For help setting up public key authentication, see the [Centos Wiki](https://wiki.centos.org/HowTos/Network/SecuringSSH#head-9c5717fe7f9bb26332c9d67571200f8c1e4324bc) 
+1. these scripts assume that you have a separate virtual drive available for the application installation at `/dev/sdc` (assuming `/dev/sda` is your main system drive and `/dev/sdb` is your swap drive).  
+1. Edit the `hosts` file in this directory and replace the ip address for your Hydra application with your server's IP
+1. Edit the `vars:` section of the `vanilla.yml` file and enter data for your system - see the commented lines
+1. Run `ansible-playbook -i hosts vanilla.yml` from the root directory of this repo on your local machine
+1. if the playbook fails, you can restart it at a particular task with `ansible-playbook -i hosts vanilla.yml --start-at-task='rolename | taskname'`
```
Comment (acozine) [inline README.md:45]: dang, I didn't see this one before either - initial cap here?

**Comment by acozine** [comment]
also, can you please squash your commits?

**Pull request merged**